tsconfig.json

```
@@ -1,11 +1,7 @@
 {
-  // see https://www.typescriptlang.org/tsconfig to better understand tsconfigs
   "include": [
-    "./src/*/",
-    "./types/index.d.ts"
-  ],
-  "exclude": [
-    "./src/stories/"
+    "src",
+    "types/index.d.ts"
```

```suggestion
    "src"
```

It can be removed if you move `types/index.d.ts` to `src/env.d.ts`